Formula/apcupsd.rb

```diff
@@ -60,7 +60,7 @@ def caveats
 
     if MacOS.version >= :el_capitan
       s += <<-EOS.undent
-        Note: On OS X El Capitan and above, the kernel extension currently
+        Note: On macOS El Capitan and above, the kernel extension currently
```

Comment:
When referring to a specific version we should use the correct naming e.g. OS X in this case.